llvm/test/CodeGen/X86/unaligned_extract_from_vector_through_stack.ll

```diff
@@ -0,0 +1,23 @@
+; RUN: llc < %s  | FileCheck %s
```

Comment:
Use -mtriple, and remove the triple/datalayout from the IR

Reply:
Done